doc/TOR.md

```diff
@@ -1,5 +1,34 @@
 HOWTO USE TOR WITH C-LIGHTNING
 
+what do we support
```

Comment:
It would be nice to give examples of how to set up each use-case here, rather than just claiming it is possible without showing how.

Reply:
yes would be nice :-)